content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -437,29 +437,30 @@ HD/Library/LaunchDaemons" file's _$JENKINS_HOME_ property.
 By default, the _$JENKINS_HOME_ will be set to "Macintosh
 HD/Users/Shared/Jenkins".
 
-**Ubuntu/Debian**
+**Linux**
 
-If you install Jenkins using a Debian package, you can find and edit the
-location of your _$JENKINS_HOME_ by editing your "/etc/default/jenkins" file.
+By default, `$JENKINS_HOME` is set to `/var/lib/jenkins`
+and `$JENKINS_WAR` is set to `/usr/share/java/jenkins.war`.
 
-By default, the _$JENKINS_HOME_ will set to "/var/lib/jenkins" and your
-$JENKINS_WAR will point to "/usr/share/jenkins/jenkins.war".
+You can edit the location of `$JENKINS_HOME`
+by adding the following to `/etc/systemd/system/jenkins.service.d/override.conf`:
```

Comment:
Possibly suggest `systemd edit jenkins` here as well?
```suggestion
by adding the following to `/etc/systemd/system/jenkins.service.d/override.conf` (for example, using `systemd edit jenkins`):
```

Reply:
Done in commit 4b280c8.